suites/pacific/iscsi/tier_1_iscsi.yaml

```diff
@@ -0,0 +1,121 @@
+tests:
```

Comment:
Can we rename this file to `tier-1_rbd_test-iscsi.yaml`? This helps align with our new filename guidelines.

Reply:
Renamed the files.